### ISSUE TITLE
The number of samples to calculate the FID?

### ISSUE THREAD

**Issue by Zzlongjuanfeng** [AlamiMejjati/plop#7]
Hello, in your link of FID implementation, the author emphasizes the number of samples to calculate the Gaussian statistics (mean and covariance) should be greater than the dimension of the coding layer. The number of the horse test set is only 120, and maybe the FID distance is not suitable here?

**Comment by AlamiMejjati** [comment]
Yup, FID is indeed not the best measure in that case. In a newer version of the paper(that will appear shortly), we replaced FID measurements by KID (Kernel Inception Distance: https://arxiv.org/pdf/1801.01401.pdf) ones.

**Comment by ShihuaHuang95** [comment]
@Zzlongjuanfeng Why use test set? Shouldn't we use training set instead?
